packages/fhs/src/main/deb/dcache.logrotate

```diff
@@ -1,7 +1,26 @@
+#Note: The following may work only for dCache logs managed by rsyslog (i.e. not for its access- and event logs, which are managed by logback)!
+
+
+
+
```

Comment:
empty lines

Reply:
Well I think such config files shouldn't be possible candidates for the IOCCC ;-)
The comment in the beginning would apply to any logrotate section thay may be added sooner or later to this file (right now only `/var/log/dcache/*.log {}`)... but I think the general goal should be to get all logs rotated by the standard systemtools (systemd/journald and/or logrotate.

So I've added some empty lines that this comment is actually noticed and to indicate it applies to all the following an not just the next section.